src/pages/ShoppingList.js

```diff
@@ -1,7 +1,43 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import fb from '../lib/firebase';
 
-const ShoppingList = () => {
-	return <h1>Hello from the Shopping List</h1>;
+
+const ShoppingList = ({ token }) => {
+	const [shoppingListItems, setShoppingListItems] = useState([]);
+
+	//Thanks for sharing your code!!
+	useEffect((token) => {
```

Comment:
this token inside `useEffect((token) => {`can be moved at the end in the brackets [ ] when closing the function. Instead of leaving the brackets empty, you can put the token there so that useEffect gets triggered when a token is available.